shared/AppInsightsCore/package.json

```diff
@@ -37,7 +38,8 @@
         "rollup-plugin-node-resolve": "^3.4.0",
         "rollup-plugin-replace": "^2.1.0",
         "rollup-plugin-uglify": "^6.0.0",
-        "tslint": "^5.11.0",
+        "tslint": "^5.19.0",
+        "tslint-config-prettier": "^1.18.0",
```

Comment:
What is this package doing?

Reply:
it makes tslint and prettier work together without conflicts: https://www.npmjs.com/package/tslint-config-prettier